source/includes/note-flexible-sync-prerequisite.rst

```diff
@@ -1,5 +1,9 @@
-.. note:: Flexible Sync Prerequisites
+.. note:: Data Source Requirements for Device Sync
 
-   Enabling Flexible Sync in Your App Services Application requires a
-   non-sharded MongoDB Atlas cluster running :manual:`MongoDB 5.0 or greater
-   </release-notes/>`
\ No newline at end of file
+   To enable Device Sync, your App Services App must have at least one linked
+   data source that meets the following requirements:
+
+   - A non-sharded MongoDB Atlas cluster running
+     :manual:`MongoDB {+sync-min-mongo-version-required+} or greater </release-notes/>`.
```

Comment:
Tiny suggestion since we're already touching this:
```suggestion
     :manual:`MongoDB {+sync-min-mongo-version-required+} or later </release-notes/>`.
```